tour/getting-started.md

```diff
@@ -2,4 +2,29 @@
 title: Getting Started
 ---
 
-Talk about how to get started.
+First make sure you have Java, Maven and Git installed on your machine.  Oh you are already rocking? OK let's go!
+
+1. Clone the tour onto your machine:
+```commandline
+git clone -b tour https://github.com/apache/accumulo-website.git tour
+cd tour
+```
+2. Open Main.java in your favorite editor.
+```commandline
+vim ./src/main/java/tour/Main.java
+```
+Notice the main method creates a MiniAccumuloCluster with a root password of "tourguide".  MiniAccumuloCluster is a mini
+version of Accumulo that runs on your local filesystem.  It should only be used for development purposes but will work
+great here on the tour.
+
+3. Modify the _exercise_ method to print a hello message. You will put your code in this method for each lesson.
+```java
+private static void exercise(MiniAccumuloCluster mac) {
+    // start writing your code here
+    System.out.println("Hello world");
```

Comment:
Could use logger here instead

Reply:
would need to ensure `mvn -q` does not suppress logging output.

Reply:
I wasn't going to mention logging since its not specific to Accumulo or needed here.  

The mvn -q won't suppress logging.  You can actually see the INFO log about MAC starting up.

Reply:
> I wasn't going to mention logging since its not specific to Accumulo or needed here.

I agree with this this.. nice to keep it simple and focused on Accumulo APIs.

Reply:
A logger doesn't need to be used but I suggested it because Accumulo still outputs log info messages.  I think it's confusing and hard to read when log messages and Sys.out.println is mixed (unless you are using Sys.out.prinln to output data)